src/website/src/pages/ec/components/button/docs/usage.mdx

```diff
@@ -9,33 +9,54 @@ import { Paragraph } from '@ecl/website-components';
   Use a button to trigger and carry out an action on a page.
 </Paragraph>
 
+## Anatomy
+
+Buttons use 3 major attributes:
+
+- Labels
+  - Should be clear and distinct.
+  - Use verbs to describe the action performed when interacting with the button.
+  - Keep it short, no more than 3 words and only capitalise the first word.
+- Colour
+  - Choose a colour that provides sufficient contrast between the label and background.
+- Icon
+  - Choose an icon that reinforce or clarify the meaning of a label in that particular context.
+
+| Name  | Mandatory | Restrictions |
+| ----- | :-------: | -----------: |
+| Label |     X     |              |
+| Icon  | centered  |              |
```

Comment:
That's not very clear for the icon. The columns says "mandatory", so what is the "centered" for?

Reply:
I have no idea how this ended up here :)